docs/ensembl-help/using-ensembl/ensembl-apps/entity-viewer/entity-viewer.md

```diff
@@ -14,30 +14,63 @@ status: draft
 
 # What is the Entity viewer?
 
-Entity viewer collates information about the gene of interest and related entities, such as its transcripts and proteins. This page is composed of the Main display, Right-hand panel, Navigation bar and Action bar.
- 
+Entity viewer reveals detailed information about an entity ie a gene, variant or location of interest and their related transcripts, proteins and functions.
+
+The Entity viewer consists of a Navigation bar, the Main display, the Right-hand panel and an Action bar.
+
+## Navigation bar
+
+The Navigation bar summarises details of an entity such as a gene, variant or location. Eg A gene summary lists the gene symbol, Ensembl stable ID, biotype, strand and genomic coordinates. 
+
+The information about an entity that can be seen in the Right-hand panel are also listed on the Navigation bar eg an overview and references are listed for a gene, this information can be seen by opening the Right-hand side panel.  
+
 ## Main display
 
-Main display presents data relating to the gene of interest. The gene summary is found at the top of the view, while data tabs including Transcripts, Gene function and Gene relationships are located beneath. Active tab is shown in black.
- 
-The gene summary section includes the gene image, Ensembl gene stable ID, gene length, strand and transcript count. Please note that the gene is always displayed in the 5’->3’ orientation, regardless of the strand. You can also go to the Genome browser from here to view your gene of interest plotted against the genome, by clicking the ‘View in Genome browser’ icon.
- 
- ## Right-hand panel
+The Main display presents detailed data relating to the entity ie a gene, variant or location of interest. 
+
+Visual summary
+A visual summary of the entity ie a gene is found at the top of the Main display. 
+
+The visual summary for a gene shows a Ensembl gene stable ID, the genes length, strand and transcript count. Exon are represented with grey boxes and introns with grey lines.
+
+Please note that the gene is always displayed in the 5’->3’ orientation, regardless of the strand. 
```

Comment:
'is always displayed in the direction of transcription regardless of the strand of the genome on which it is located'.

Reply:
done